roles/edpm_nova/tasks/configure_compute.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Configure compute configs
+  block:
+    - when: edpm_nova_cpu_shared_set|string
+      name: Set CPU shared set
+      community.general.ini_file:
+        path: "{{ edpm_nova_config_dest }}/03_compute_config.conf"
+        setype: container_file_t
+        mode: "0644"
+        create: true
+        section: "compute"
+        option: "cpu_shared_set"
+        value: "{{ edpm_nova_cpu_shared_set }}"
+
+    - when: edpm_nova_cpu_dedicated_set|string
+      name: Set CPU dedicated set
+      community.general.ini_file:
+        path: "{{ edpm_nova_config_dest }}/03_compute_config.conf"
+        setype: container_file_t
+        mode: "0644"
+        create: true
+        section: "compute"
+        option: "cpu_dedicated_set"
+        value: "{{ edpm_nova_cpu_dedicated_set }}"
+
+    - when: edpm_nova_reserved_host_memory|int > 0
+      name: Set reserved host memory
+      community.general.ini_file:
+        path: "{{ edpm_nova_config_dest }}/03_compute_config.conf"
+        setype: container_file_t
+        mode: "0644"
+        create: true
+        section: "compute"
+        option: "reserved_host_memory_mb"
```

Comment:
this is in the Default section and it is not enough to reserve memory when using CPU pinnign